<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>main_window</class>
    <widget class="QMainWindow" name="main_window">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>655</width>
                <height>430</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>655</width>
                <height>430</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>1000</width>
                <height>600</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>PlagDef</string>
        </property>
        <property name="windowIcon">
            <iconset resource="../res/resources.qrc">
                <normaloff>:/icon.ico</normaloff>:/icon.ico
            </iconset>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: rgb(77, 80, 87);</string>
        </property>
        <widget class="QWidget" name="main_widget">
            <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                </sizepolicy>
            </property>
            <property name="minimumSize">
                <size>
                    <width>0</width>
                    <height>0</height>
                </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <spacer name="top_spacer">
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeType">
                            <enum>QSizePolicy::Minimum</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>20</width>
                                <height>30</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item>
                    <widget class="QLabel" name="logo_label">
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                            </sizepolicy>
                        </property>
                        <property name="minimumSize">
                            <size>
                                <width>0</width>
                                <height>43</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>16777215</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="text">
                            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <spacer name="logo_stacked_widget_spacer">
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeType">
                            <enum>QSizePolicy::MinimumExpanding</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>20</width>
                                <height>40</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item>
                    <widget class="QStackedWidget" name="stacked_widget"/>
                </item>
                <item>
                    <spacer name="bottom_spacer">
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>20</width>
                                <height>40</height>
                            </size>
                        </property>
                    </spacer>
                </item>
            </layout>
        </widget>
    </widget>
    <resources>
        <include location="../res/resources.qrc"/>
    </resources>
    <connections/>
</ui>
